src/components/cylc/gscan/GScan.vue

```diff
@@ -128,9 +134,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                     />
                     <v-tooltip top>
                       <template v-slot:activator="{ on }">
-                        <span v-on="on">{{ scope.node.node.name || scope.node.id }}</span>
+                        <span v-on="on">{{ scope.node.name }}</span>
```

Comment:
Display name in the tree. Note that this is not the GraphQL node name (i.e. `node.node.name`), but the `.name` property we have when creating the hierarchy.

For `folder/five/run1`, its name is `folder/five/run1`, but we have just `run1` to be used when displaying it. In the same manner, when we are adding the workflow-name-part `folder`, its `.name` will be... `folder` :+1: I accidentally modified the `node.node.name` changing its value, and when the deltas updated it in development I had a nasty bug :bug: took me near 1 hour to realize the mistake I had made :smile: